splinepy/spline.py

```diff
@@ -1118,6 +1118,33 @@ def coordinate_references(self):
 
         return self._data["coordinate_references"]
 
+    @property
+    def multi_index(self):
+        """
+        Easy control point / weights access using (unraveled) multi index.
```

Comment:
That is dangerous, as multi-index changes with simple modifications, such as order-elevation and knot insertion

Reply:
why exactly is it dangerous?

Reply:
If someone accesses `multi_index` then performs order elevation the old multi_index is no longer valid

Reply:
yes, that's why modification calls deletes all the saved data